repository-data/webfiles/src/main/resources/site/freemarker/ndrs/icon-library-item.ftl

```diff
@@ -0,0 +1,169 @@
+<#ftl output_format="HTML">
```

Comment:
I am interested to know why you need this in NDRS. Does NDRS have its own Icon Library?